spec/services/people/importer_spec.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe People::Importer do
+  subject(:importer) { described_class.new(input_data) }
+
+  let(:input_data) do
+    {
+      prison_number: 'G3239GV',
+      last_name: 'ABBELLA',
+      first_name: 'AVEILKE',
+      middle_names: 'EMMANDA',
+      date_of_birth: '1965-10-15',
+      aliases: nil,
+      pnc_number: '82/18053V',
+      cro_number: '018053/82G',
+      gender: 'F',
+      ethnicity: 'White: Eng./Welsh/Scot./N.Irish/British',
+      nationalities: 'British'
+    }
+  end
+
+  let!(:ethnicity) { create(:ethnicity, title: 'White: Eng./Welsh/Scot./N.Irish/British') }
+  let!(:gender) { create(:gender, nomis_code: 'F') }
```

Comment:
Can we assume that there is a one-one mapping between gender codes coming out of NOMIS with those in our system?

Reply:
Same as above